MiniAn/minian_definitions.py

```diff
@@ -0,0 +1,126 @@
+
+class Messages:
+    """
+    Messages
+    """
+
+    class Main:
```

Comment:
I prefer how it was before

Reply:
When you create classes or namespaces like that, try to separate names by their meaning, not by the location in the code and python names of variables (e.g. dictionary, keys)